Data/TeleporterRegions.xml

```diff
@@ -796,6 +796,11 @@
 		<tiles from="(5573, 629, 42)" frommap="Trammel" to="(5503, 570, 51)" tomap="Trammel" />
 		<tiles from="(5573, 630, 42)" frommap="Trammel" to="(5503, 571, 51)" tomap="Trammel" />
 	</Teleporter>
+    <Teleporter>
```

Comment:
https://github.com/TrueUO/TrueUO/blob/master/Scripts/Items/Internal/DespiseTeleporter.cs

The DespiseTeleporters being replaced here specifically make sure you are not trying to bring out a Despise creature. Replacing them would remove the other functionality in the DespiseTeleporter.cs